src/main/java/com/autotune/utils/KruizeConstants.java

```diff
@@ -395,4 +395,55 @@ public static final class KRUIZE_CONFIG_ENV_NAME {
         public static final String SETTINGS_HIBERNATE_SHOW_SQL = "hibernate_showsql";
         public static final String SETTINGS_HIBERNATE_TIME_ZONE = "hibernate_timezone";
     }
+
+    public static final class RecommendationEngineConstants {
+        private RecommendationEngineConstants() {
+
+        }
+
+        public static final class DurationBasedEngine {
+            private DurationBasedEngine() {
+
+            }
+
+            public static final class DurationAmount {
+                private DurationAmount() {
+
+                }
+
+                public static final int SHORT_TERM_DURATION_DAYS = 1;
+                public static final int MEDIUM_TERM_DURATION_DAYS = 7;
+                public static final int LONG_TERM_DURATION_DAYS = 15;
+            }
+
+            public static final class RecommendationDurationRanges {
+                private RecommendationDurationRanges() {
+
+                }
+
+                private static final double BUFFER_VALUE_IN_MINS = 0.5;
```

Comment:
We dont need a new parameter here, you can use `MEASUREMENT_DURATION_THRESHOLD_SECONDS` itself which should be the same value (30 seconds).